### PR TITLE
Don't set a dagster/solid_selection tag when you materialize a set of assets

### PR DESCRIPTION
**Pull request by gibsondan**
Summary:
Much like https://github.com/dagster-io/dagster/pull/13394 - this dagster/solid_selection (or dagster/op_selection) tag can be very large, but I can find no place where we actually use it anymore on the backend.

It still could be useful for non-asset jobs to see what ops you launched by showing the query - so instead, for each of the other callsites where we were setting a solidSelectionQuery, set that tags there instead.

Test Plan:
Load an asset job locally, press "materialize all" in the launchpad - all assets materialize but the giant tag is no longer there
In a non-asset job, materailize an op subset - tag is still there on the resulting job
Presets are no longer possible to set, but do the same for a partitioned-job that sets an op subset (is this still a thing?) and verify the tag is still set on the resulting job

### CODE REVIEW

File: js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx [hunk 1]
```diff
@@ -429,15 +440,19 @@ const LaunchpadSession: React.FC<LaunchpadSessionProps> = (props) => {
 
   const onSelectPreset = async (preset: Preset) => {
```
Comment (gibsondan) [inline js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx:441]: @smackesey is this even a codepath that you can hit in dagit anymore

**Comment by OwenKephart** [comment]
I don't feel confident enough to stamp the frontend changes, but I want to log my confirmation of the fact that the op_selection tag is not used anywhere in the backend, as all op selection information is passed via the, well, `op_selection` parameter of a JobSubsetSelector. I believe this is just a relic of a time long ago.

File: js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx (hunk 1 above)
Comment (bengotow) [reply js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx:441]: Hmm I'm not sure about this one -- I think the question is whether `pipeline.presets[]` is ever populated in the GraphQL layer anymore. if it's 100% deprecated I'd love to clean this code up more broadly.

**Review by bengotow — APPROVED** [review]
This looks good to me! As far as I'm aware there is no need for the DagsterTag.SolidSelection tag (displayed as op_selection) on asset runs at all. 🙌

File: js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx (hunk 1 above)
Comment (smackesey) [reply js_modules/dagster-ui/packages/ui-core/src/launchpad/LaunchpadSession.tsx:441]: according to tsserver `onSelectPreset` is not referenced anywhere

**Review by gibsondan — COMMENTED** [review]
hm one downside of this that I noticed while testing is that the tag now shows up in the tag editor and can be removed/edited/etc, which is a little odd

**Comment by gibsondan** [comment]
Ended up doing a much simpler fix here in https://github.com/dagster-io/dagster/pull/16676

**Pull request closed without merging**